packages/core/src/routes/logto-config.ts

```diff
@@ -182,4 +210,42 @@ export default function logtoConfigRoutes<T extends AuthedRouter>(
       return next();
     }
   );
+
+  router.put(
+    '/configs/jwt-customizer/:tokenTypePath',
+    koaGuard({
+      params: z.object({
+        tokenTypePath: z.nativeEnum(LogtoJwtTokenPath),
+      }),
+      /**
+       * Use `z.unknown()` to guard the request body as a JSON object, since the actual guard depends
+       * on the `tokenTypePath` and we can not get the value of `tokenTypePath` before parsing the request body,
+       * we will do more specific guard as long as we can get the value of `tokenTypePath`.
+       *
+       * Should specify `body` in koaGuard, otherwise the request body is not accessible even via `ctx.request.body`.
+       */
+      body: z.unknown(),
```

Comment:
can remove these?

Reply:
We can't, see the comments. If we do not explicitly define `body` guard, we can not access `body` data in the following API operation handler.

Reply:
use koaBody.

Reply:
updated

Reply:
use `koaBody()` brings trouble to the swagger file parser. Keep it as it is?
<img width="963" alt="image" src="https://github.com/logto-io/logto/assets/15182327/d868d55f-f174-4d55-a30f-650cea920416">

Reply:
See LOG-8376 to fix this later.